README.md

```diff
@@ -242,7 +247,7 @@ Pyctuator leverages Python's builtin `logging` framework and allows controlling
  
 Note that in order to control uvicorn's log level, you need to provide a logger object when instantiating it. For example:
 ```python
-myFastAPIServer = Server(
+myFastAPIServer = Server(4
```

Comment:
```suggestion
myFastAPIServer = Server(
```

Reply:
Typo- I guess

Reply:
Yup. One day we need to make the pipeline run the code in `README.md`...
Thanks